src/applications/vaos/components/FacilityHours.jsx

```diff
@@ -0,0 +1,99 @@
+// Dependencies
+import React from 'react';
+import { get } from 'lodash';
```

Comment:
I think you could remove the use of `get` in this file, we don't really need it now with optional chaining.